templates/types/extractor/fastapi/app/ui/components/upload.py

```diff
@@ -0,0 +1,86 @@
+import os
+from typing import List
+
+import reflex as rx
+from app.engine.generate import generate_datasource
+
+
+class UploadedFile(rx.Base):
+    file_name: str
+    size: int
+
+
+class UploadedFilesState(rx.State):
+    _uploaded_dir = "data"
```

Comment:
as far as i understand it, this is uploading to data (for all users) - so why is generate.py setting `private` metadata?